osmaxx/excerptexport/migrations/0022_transfer_output_files_from_extraction_orders_to_exports.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2016-04-26 14:19
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def set_export_fk_from_extraction_order_and_content_type(apps, schema_editor):
+    OutputFile = apps.get_model('excerptexport', 'OutputFile')  # noqa
+    for output_file in OutputFile.objects.all():
+        output_file.export = output_file.extraction_order.exports.get(file_format=output_file.content_type)
+        output_file.save()
```

Comment:
Doable, yes, feasible, not so much, otherwise you wouldn't be asking, right?

If you wanted to test this, extract it to a function and test that one, probably easier than getting the data (in the sense of how much time it needs).